### PR TITLE
Refactored Broadcasts

### PR DESCRIPTION
**Pull request by nehaparmar99**
## Issue that this pull request solves
Made Under DWOC and SWOC
 Closes: #372 

### Brief description of what is fixed or changed
 Made a test_data directory under frontend/src folder. Make a file named broadcasttext.js and added the broadcast data there.This json is rendered on the broadcast and all_broadcast page and also in the carousel

_Put an `x` in the boxes that apply_

- [x] New feature (non-breaking change which adds functionality)

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

### CODE REVIEW

**Comment by nehaparmar99** [comment]
@Kajol-Kumari review

**Review by tejan-singh — APPROVED** [review]
LGTM 👍🏻

File: frontend/src/components/broadcast/Carousel/Carousel.js [hunk 1]
```diff
@@ -5,40 +5,14 @@ import "owl.carousel/dist/assets/owl.theme.default.css";
...
-                tech projects (externships) from exciting startups ✅ Interact
-                with the biggest names in India's tech industry ✅ Participate
-                in engaging Masterclass sessions from top engineers in the
-                country ✅Get discovered by Crio's hiring partners Apply now ❗
-                👇 https://www.crio.do/crio-winter-of-doing/ Limited slots are
-                available ❗❗ Thank you, With regards, HITK Tech Community👩‍💻👨‍💻
-                (By HITK students,For HITK students)`;
+import { dataa } from "../../../test_data/broadcasttext";
+export default function Owl({ head, cards }) {
+  let s1t = dataa[0].title;
```
Comment (salil-naik) [inline frontend/src/components/broadcast/Carousel/Carousel.js:10]: Variable names should be self-descriptive as much as possible. Writing variables like s1t, s1d, etc is a bad practice.

**Review by salil-naik — CHANGES_REQUESTED** [review]
Hi @nehaparmar99 . Good work so far. 

Please do the following changes. 
- [ ] Use descriptive variable names

- [ ] use JSON file to store the data.

**Comment by nehaparmar99** [comment]
@salil-naik made both the changes. please check

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:

**Pull request merged**